pkg/scheduler/scheduler_test.go

```diff
@@ -870,6 +876,19 @@ func Test_UnionedGVKs(t *testing.T) {
 				framework.StorageClass:          framework.All - framework.Delete,
 			},
 		},
+		{
+			name: "mock EventsToRegister plugin for test EventsToRegister method returns error",
```

Comment:
```suggestion
			name: "one EventsToRegister returns an error",
```

Reply:
And, can we move this error test scenario into `Test_buildQueueingHintMap` rather? 
`Test_UnionedGVKs` is a test focusing on UnionedGVKs, as the name tells.

Reply:
oh, my bad. fixed